<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewSchedule</class>
 <widget class="QDialog" name="NewSchedule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Schedule</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>420</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Schedule/Course Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_NameSchedule">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>210</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>The name of the schedule or course</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Date:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>End Date:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_Start">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_End">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of Students:</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_StartDate">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Use a calendar to select the date</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit Student Names:</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_EndDate">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Use a calendar to select the date</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_StudentNames">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Maximum Shifts:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Minimum Shifts:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit Shifts:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of Shifts:</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_ShiftEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>250</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Edit the shifts</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Directory:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_DirectorySave">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>280</y>
     <width>210</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>The directory/folder where the schedule file (.schd) will be saved</string>
   </property>
   <property name="text">
    <string>C:\EM_Scheduler\schedules</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Export Directory:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_DirectoryExport">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>310</y>
     <width>210</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>The directory/folder where the spreadsheet (.xls) will be stored</string>
   </property>
   <property name="text">
    <string>C:\EM_Scheduler\spreadsheets</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_AdvMode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>100</width>
     <height>19</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Extended options will be provided such as editing shifts</string>
   </property>
   <property name="text">
    <string>Advanced Mode</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_NumStudents">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>43</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_NumMaxShifts">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>43</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_NumMinShifts">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>43</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_NumShifts">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>43</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewSchedule</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewSchedule</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
